DUI3-DX/Connectors/ArcGIS/Speckle.Connectors.ArcGIS3/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,54 @@
+/*
+
+   Copyright 2022 Esri
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       https://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+*/
+using System.Reflection;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("ConnectorArcGIS")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("Acme")]
+[assembly: AssemblyProduct("ConnectorArcGIS")]
+[assembly: AssemblyCopyright("Copyright © Acme 2021")]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCulture("")]
+
+// Setting ComVisible to false makes the types in this assembly not visible
+// to COM components.  If you need to access a type in this assembly from
+// COM, set the ComVisible attribute to true on that type.
+[assembly: ComVisible(false)]
+
+// The following GUID is for the ID of the typelib if this project is exposed to COM
+[assembly: Guid("D2D8A42A-F983-4814-AF7C-4E4C0609FA80")]
+
+// Version information for an assembly consists of the following four values:
+//
+//      Major Version
+//      Minor Version
+//      Build Number
+//      Revision
+//
+// You can specify all the values or you can default the Build and Revision Numbers
+// by using the '*' as shown below:
+// [assembly: AssemblyVersion("1.0.*")]
+// TODO: Fix static version assignee
+[assembly: AssemblyVersion("2.0.999.0")]
```

Comment:
This was mine previous hardcoded `AssemblyVersion`, this should be ideally retrieved from the build targets? Not sure, it wasn't working properly before thats why I hard coded it otherwise add-in was crashing on ArcGIS. @JR-Morgan any input on this why this info couldn't retrieved from targets before?

Reply:
We don't need the assembly info anyway. I've deleted it